cfpb/cfgov-refresh
# Post Preview Snapshot Organism

Adds the snapshot organism that pulls in posts from a descendant Browsefilterable page of a Sublanding page. The content editors choose the limit of posts to show, and the page will spawn the specified number of posts with the Post Preview organism as the markup.

This will only exist on the Notice and Comment page as of now.

### Test
- Run `./cfgov/manage.py migrate`
- Go to a Sublanding page
- Select Post Preview Snapshot in the content streamfield
```
- ***Important Note***Page Hierarchy
   Sublanding Page 
             | -->  Browse Filterable Page 
                                  |--> list of pages (Document Detail/ Learn Page).
```
- See if the list pops up.

@kave 
@richaagarwal
